static/coffee/locations.coffee

```diff
@@ -116,9 +123,12 @@ initMap = (id, geojson, question, districtLabel) ->
     info.update()
 
   clickFeature = (e) ->
-    if (allowDistrictZoom and e.target.feature.properties.level == 1)
+    if ([1,2].indexOf(e.target.feature.properties.level) != -1)
```

Comment:
in Coffeescript, you can directly write `e.target.feature.properties.level in [1, 2]`. I suggest you to install the Coffeescript command line via `npm install coffee-script` and try it for yourself:
```
coffee> 1 in [1, 2]
true
```